src/traces/pie/attributes.js

```diff
@@ -181,6 +181,21 @@ module.exports = {
     textfont: extendFlat({}, textFontAttrs, {
         description: 'Sets the font used for `textinfo`.'
     }),
+    insidetextorientation: {
+        valType: 'enumerated',
+        role: 'info',
+        values: ['h', 'r', 't', 'auto'],
```

Comment:
Hmm. I'd vote for `'horizontal', 'rotational'`, `'tangential'`, `'auto'`. I see your point for `'h'` making it on-par with the typical `orientation` values, but values `'r'` and `'t'` look pretty obscure to me.

Reply:
`rotational` or `radial`?
Also to note something like `insidetextorientation: 'tangential'` is hard to read & write.

Reply:
yep, I meant to type `'radial'`. Edited above.

Reply:
Changed in https://github.com/plotly/plotly.js/pull/4420/commits/a27dc29ca87303fd53bf994bfe1203b3c5e96887.

Reply:
Thank you very much!